example_test.go

```diff
@@ -103,72 +104,60 @@ func Example() {
 		return
 	}
 
+	// Ping a Tarantool instance to check connection
 	resp, err := client.Ping()
 	fmt.Println("Ping Code", resp.Code)
 	fmt.Println("Ping Data", resp.Data)
 	fmt.Println("Ping Error", err)
 
-	// delete tuple for cleaning
+	// Delete tuples with primary key { 10 } and { 11 }
 	client.Delete(spaceNo, indexNo, []interface{}{uint(10)})
-	client.Delete(spaceNo, indexNo, []interface{}{uint(11)})
+	// the same:
+	client.Delete("test", "primary", []interface{}{uint(11)})
 
-	// insert new tuple { 10, 1 }
+	// Insert a new tuple { 10, 1 }
 	resp, err = client.Insert(spaceNo, []interface{}{uint(10), "test", "one"})
-	fmt.Println("Insert Error", err)
```

Comment:
Why did you removed them? The example helps to understand how we may work with responses (at least how we could marshal them).

Reply:
AFAIR we discussed it with @Totktonada (or may be it was you?) that it is a duplicate code and it is enough to show that every method returns resp and err only once.